pkg/cmd/openshift-kube-apiserver/openshiftkubeapiserver/patch.go

```diff
@@ -239,11 +248,15 @@ func (i *KubeAPIServerInformers) GetOpenshiftSecurityInformers() securityv1infor
 func (i *KubeAPIServerInformers) GetOpenshiftUserInformers() userinformer.SharedInformerFactory {
 	return i.OpenshiftUserInformers
 }
+func (i *KubeAPIServerInformers) GetOpenshiftAuthzInformers() authzinformer.SharedInformerFactory {
```

Comment:
https://github.com/openshift/origin/pull/22783/files#diff-01a8de689f656fdb0f202987ee6c1f2cR117   ok? They're namespaced so the 
openshift/client-go/authorization/informers/externalversions/authorization/v1/  RoleBindingRestrictionInformer won't work here (I might be missing something, might be unclear on what you're asking here)

Reply:
We cannot use this authorization informer for anything else because it is JSON based.  Hence it needs to be RBR only.  Something like:

`func (i *KubeAPIServerInformers) GetOpenshiftRoleBindingRestrictionInformer() RoleBindingRestrictionInformer {`

Reply:
done